app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -208,6 +208,7 @@ class BookmarkFragment : LibraryPageFragment<BookmarkNode>(), UserInteractionHan
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         return when (item.itemId) {
             R.id.bookmark_search -> {
+                BookmarksManagement.searchIconTapped.record(NoExtras())
```

Comment:
should this metric be moved into `bookmarkInteractor.onSearch()` so it can additionally be covered with a test?

Reply:
That's a good idea.  I'll update.  Thanks